Submitted By: Nick Fotopoulos (weasel at beyondnormal dot org)
Date: 2004-05-20
Initial Package Version: 2.0.0
Upstream Status: None
Origin: Self
Description: Fixes incorrect function parameter type declaration in inital_region

diff -Naur gimp-2.0.0/app/paint-funcs/paint-funcs.h gimp-2.0.0-fixed/app/paint-funcs/paint-funcs.h
--- gimp-2.0.0/app/paint-funcs/paint-funcs.h	2004-03-10 08:13:43.000000000 -0500
+++ gimp-2.0.0-fixed/app/paint-funcs/paint-funcs.h	2004-05-20 16:31:15.200108400 -0400
@@ -356,7 +356,7 @@
 					   gint,
                                            GimpLayerModeEffects,
                                            gboolean *,
-                                           CombinationMode);
+                                           InitialMode);
 
 void  combine_regions                     (PixelRegion *, PixelRegion *,
 					   PixelRegion *, PixelRegion *,
